src/components/partials/profile/SendTokenModal.vue

```diff
@@ -116,38 +125,49 @@ export default class SendTokenModal extends Vue {
     tokenAmount = "";
     busy = false;
     error = "";
-    selectedAccount: string | undefined = "";
+    selectedAccount = "";
     web3Accounts!: InjectedAccountWithMeta[];
-
-    created() {
-        if (this.web3Accounts.length === 1) {
-            this.selectedAccount = this.web3Accounts[0].address;
-        }
-    }
+    store = useStore();
 
     onSelectAccount(event: Event) {
         const target = event.target as HTMLTextAreaElement;
         this.selectedAccount = target.value;
     }
 
-    sendToken() {
+    async sendToken() {
         this.error = "";
-        if (!this.validateInput()) {
-            this.error = "Please insert a positive float as token amount.";
-        } else {
-            alert("This Feature will be added soon :)");
+        if (this.validate()) {
+            try {
+                await this.store.dispatch("SEND_TOKEN", {
+                    chain: this.identity.chain?.toLowerCase(),
+                    senderAddress: this.selectedAccount,
+                    receiverAdress: this.identity.basicInfo.address,
```

Comment:
you have a typo in receiverAddress. That would cause the transaction to be send to the wrong account!